cpp/src/arrow/testing/random_test.cc

```diff
@@ -240,6 +240,7 @@ class RandomDecimalArrayTest : public ::testing::Test {
     ASSERT_LE(over_half, non_nulls * 0.7);
     ASSERT_GE(negative, non_nulls * 0.3);
     ASSERT_LE(negative, non_nulls * 0.7);
+    ASSERT_OK(values.ValidateFull());
```

Comment:
There's already a `ASSERT_OK(array.ValidateFull());` at the beginning, this doesn't seem to add anything?